Makefile

```diff
@@ -55,18 +55,23 @@ docker-compose.override.yml: ## Create docker compose override file
 docker-compose.anchore.yml: ## Create docker compose override file with anchore
 	cp docker-compose.anchore.yml.dist docker-compose.override.yml
 
+create-docker-dirs: ## Create .docker directories with your user
```

Comment:
Is this actually necessary? Isn't the user setting enough?

Reply:
In my experience, it is not enough. Even though I was using my own user inside some of the docker containers, the `.docker` directory (and folders under it) was owned by root after creation. This caused `permission denied` errors inside containers where my local Linux user was used. That's why I used this solution so that these dirs will be owned by my Linux user and hence can operate on their mounts inside the containers as well.

Reply:
I would probably add these commands to the start target, before the compose up part. This target will never be executed on it's own.

Also, a shorter form: `mkdir -p .docker/volumes/{mysql,vault/file,vault/keys}`

Reply:
This is needed for the `anchorestart` target as well on Linux, that's why created this as a new target to avoid code duplication. In this case, do you think it's reasonable like this? (I'll commit the shorter `mkdir` form, thx!)

Reply:
anchorestart will be removed soon, but I don't want to block this PR, so I will just simplify the make targets once I get there.